spec/index.html

```diff
@@ -1286,15 +1286,16 @@ <h3>Identifiers</h3>
     <dd>
 <p>Literals can only act as object nodes.</p>
 
-    <p><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a> always have a datatype. Language-tagged strings get
+    <p><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a> always have a datatype.
+    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">Language-tagged strings</a> get
     the datatype <code>rdf:langString</code>. When there is no
     language tag or datatype specified the literal is assumed to have the datatype
     <code>xsd:string</code>. </p>
```

Comment:
It's not merely assumed, as `"a"` is supposed to match `"a"^^xsd:string` and vice versa.
```suggestion
    language tag or datatype specified, the literal is treated as if the datatype
    <code>xsd:string</code> was specified. </p>
```